docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -272,11 +272,9 @@ You should be able to move the custom styles to the root class key.
 
 - [ExpansionPanelActions] Rename the `action` CSS class `spacing`.
 
-### Selection controls
+### Switch
 
-- [Switch][Radio][Checkbox] Improve specification compliance.
-
-  Refactore the implementation to make it easier to override the styles.
+- [Switch] Refactore the implementation to make it easier to override the styles.
```

Comment:
I have reverted the specificity increase. You can see the impact on the customization examples. I believe the Radio and Checkbox changes are no longer breaking.